Azure/sonic-buildimage
# [sonic-thermalctld] Depend on sonic-platform-common

#### Why I did it

Unit tests for thermalctld depend on sonic-platform-common as of https://github.com/Azure/sonic-platform-daemons/pull/157

#### How I did it

Add dependency in sonic-thermalctld.mk makefile.

#### How to verify it

Ensure build passes once https://github.com/Azure/sonic-platform-daemons/pull/157 is included in a submodule update.

#### Which release branch to backport (provide reason below if selected)
